cognite/neat/rules/_analysis/_information_rules.py

```diff
@@ -315,19 +322,23 @@ def symmetrically_connected_classes(
                 sym_pairs.add((source, target))
         return sym_pairs
 
-    def as_property_dict(
-        self,
-    ) -> dict[str, list[InformationProperty]]:
+    def as_property_dict(self, use_reference: bool = False) -> dict[str, list[InformationProperty]]:
```

Comment:
Isn't this handled in by `InformationArchitectRulesAnalysis(rules.reference).as_property_dict()`?
```suggestion
    def as_property_dict(self) -> dict[str, list[InformationProperty]]:
```